pkg/controller/add_backup.go

```diff
@@ -0,0 +1,7 @@
+package controller
+
+func init() {
+	// AddToManagerFuncs is a list of functions to create controllers and add them to a manager.
+	// TODO: Uncomment this line once card CO-488 is completed (adding HiveConfig option for this).
```

Comment:
Are you planning to complete `CO-488` in this PR or in a subsequent PR?

Reply:
Subsequent PR.